region/maridia/outer/Mama Turtle Room.json

```
@@ -955,44 +1020,88 @@
       "requires": [
         "canPreciseGrapple",
         "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Grapple, Mama Turtle is Sleeping",
+      "requires": [
+        "canUseGrapple",
+        {"obstaclesNotCleared": ["A"]}
       ],
-      "devNote": "Need to consider other transit options: springball, HBJ."
+      "clearsObstacles": ["B"],
+      "note": "Grappling the block does not require canPreciseGrapple if Mama Turtle is asleep, as there is no risk of death on a miss."
     },
     {
-      "link": [6, 3],
-      "name": "Base",
+      "link": [6, 2],
+      "name": "Tricky Speed HiJump",
       "requires": [
-        {"or": [
-          "canHeroShot",
-          "h_canUsePowerBombs",
-          "Spazer",
-          "Wave"
-        ]}
+        "HiJump",
+        "canTrickyJump",
+        "SpeedBooster",
+        "canWalljump"
+      ],
+      "clearsObstacles": ["B"],
+      "note": [
+        "Briefly run on the top ledge on either side (left is slightly easier) then jump and wall jump on the grapple block."
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Tricky Speed HiJump, Walljumpless",
+      "requires": [
+        "h_canBackIntoCorner",
+        "HiJump",
+        "canTrickyJump",
```

```suggestion
        "canInsaneJump",
```
100% doesn't use this as a frame saver in the room because the jump isn't reliable.

---

If it's this difficult, I'm thinking of leaving the strat off and they can just get back on Mama Turtle and get to the top left